test/CMakeLists.txt

```diff
@@ -297,6 +298,7 @@ target_include_directories(
   tiledb_unit BEFORE PRIVATE
     ${TILEDB_CORE_INCLUDE_DIR}
     ${TILEDB_EXPORT_HEADER_DIR}
+    ${TILEDB_SOURCE_ROOT}/test/support
```

Comment:
This directory should be an interface property on the test support library, so that it will be incorporated into the include path in all targets that link to it, not just `tiledb_unit`.

Reply:
seems not needed here, removed